.github/workflows/post-merge-tests.yml

```diff
@@ -47,3 +47,114 @@ jobs:
     - name: Test Cantera
       run:
         python3 `which scons` test show_long_tests=yes verbose_tests=yes --debug=time
+
+  windows-mingw:
+    name: mingw on Windows, Python 3.10
+    runs-on: windows-2022
+    timeout-minutes: 120 # MinGW is slooooow
```

Comment:
I think I'd prefer to keep the `mingw` run as part of the normal CI. It does occasionally catch some odd issues, and despite the comment here, these CI runs are actually faster than the MSVC ones in many cases.  There must have been some updates to the MinGW packages that helped with compilation time.

Reply:
No problem - dropped that commit.